### ISSUE TITLE
Code folding triangles don't update on unfold when block above them is removed

### ISSUE THREAD

**Issue by jwestbrook** [adobe/brackets#12456]
Scenario : 

In a PHP file with mixed HTML and PHP I'm folding the `<tr>`s of a table. If I have a set of 3 rows and I unfold row 2 which has 2 PHP blocks within it, and remove one of the blocks, then go to row 3 to unfold - the row will unfold but the triangle will not update and I think the folded state will not save.

For example

Step 1
```html
<table>
<tr>...</tr>
<tr>...</tr>
<tr>...</tr>
</table>
```

Step 2
```html
<table>
<tr>...</tr>
<tr>
    <td>
       <?php
       if($variable)
       {
           foreach($variable as $v)
           {

           }
       }
       ?>
   </td>
</tr>
<tr>...</tr>
</table>
```

Step 3
```html
<table>
<tr>...</tr>
<tr>
    <td>
       <?php
       if($variable) //remove
       {                  //remove
           foreach($variable as $v)
           {

           }
       }             //remove
       ?>
   </td>
</tr>
<tr>...</tr>
</table>
```

Step 4
```html
<table>
<tr>...</tr>
<tr>
    <td>
       <?php
           foreach($variable as $v)
           {

           }
       ?>
   </td>
</tr>
<tr>...</tr>
</table>
```
At this point unfolding row 3 will show the code contained, but the triangle will not rotate and will not respond to any more clicks unless you reload Brackets.

OSX 10.11.5

Release 1.6 build 1.6.0-16680 (release 3df0ac6fa) 
build timestamp: Tue Jan 12 2016 06:18:55 GMT-0800

Don't know if its worked before

I've attempted this with and without extensions enabled.

No errors logged in console

**Comment by thehogfather** [comment]
@jwestbrook thanks for this. I'm able to reproduce this with some small variations on your test.
I am fairly sure this is related to #12232

**Comment by jwestbrook** [comment]
Yep that looks like the same behavior I get - glad I could illustrate a test case

**Comment by ficristo** [comment]
Should be fixed in 1.8 by https://github.com/adobe/brackets/pull/12521
